.editorconfig

```diff
@@ -0,0 +1,18 @@
+# https://EditorConfig.org
+
+# top-most EditorConfig file
+root = true
+
+[*]
+charset = utf-8
+end_of_line = lf
+indent_size = 2
```

Comment:
```suggestion
```

Based on the EditorConfig website, when `indent_style` is set to `tab`, it may be desirable to leave `indent_size` unspecified so readers may view the file using their preferred indentation width, so I suggest removing this setting here.

Reply:
Good call. I only use spaces in other projects, so didn't know. Thanks :)